includes/class.llms.session.php

```diff
@@ -263,6 +263,39 @@ protected function maybe_update_id() {
 
 	}
 
+	/**
+	 * Determines if the cookie and related save/destroy handler actions should be initialized
+	 *
+	 * When doing CRON or when on the admin panel we don't want to load, otherwise we do.
+	 *
+	 * @since [version]
+	 *
+	 * @return boolean
+	 */
+	protected function should_init() {
+
+		// Load by default in most places.
+		$init = true;
+
+		if ( defined( 'DOING_CRON' ) && DOING_CRON ) {
+			// Don't init during crons.
+			$init = false;
+		} elseif ( is_admin() && ! wp_doing_ajax() ) {
+			// Don't init on the admin panel unless it's an ajax request.
+			$init = false;
+		}
```

Comment:
```suggestion
		// Don't init during crons or on the admin panel unless it's an ajax request.
		if ( ( defined( 'DOING_CRON' ) && DOING_CRON ) || ( is_admin() && ! wp_doing_ajax() ) {
			$init = false;
		}
```

Reply:
oh you want to play code golf?

Reply:
I don't even know what that means, mr. country club :P